sdk/dapp-kit/src/components/connect-modal/ConnectModal.tsx

```diff
@@ -68,54 +69,59 @@ export function ConnectModal({ trigger }: ConnectModalProps) {
 
 	return (
 		<Dialog.Root open={isConnectModalOpen} onOpenChange={onOpenChange}>
-			<Dialog.Trigger className={styles.triggerButton}>{trigger}</Dialog.Trigger>
+			<StyleMarker>
+				<Dialog.Trigger className={styles.triggerButton}>{trigger}</Dialog.Trigger>
+			</StyleMarker>
 			<Dialog.Portal>
-				<Dialog.Overlay className={styles.overlay} />
-				<Dialog.Content className={styles.content} aria-describedby={undefined}>
-					<div
-						className={clsx(styles.walletListContainer, {
-							[styles.walletListContainerWithViewSelected]: !!currentView,
-						})}
-					>
-						<div className={styles.walletListContent}>
-							<Dialog.Title className={styles.title}>Connect a Wallet</Dialog.Title>
-							<WalletList
-								selectedWalletName={selectedWallet?.name}
-								onPlaceholderClick={() => setCurrentView('getting-started')}
-								onSelect={(wallet) => {
-									setSelectedWallet(wallet);
-									connectWallet(wallet);
-								}}
-							/>
-						</div>
-						<button
-							className={styles.whatIsAWalletButton}
-							onClick={() => setCurrentView('what-is-a-wallet')}
-							type="button"
-						>
-							What is a Wallet?
-						</button>
-					</div>
+				<StyleMarker>
+					<Dialog.Overlay className={styles.overlay}>
```

Comment:
I nested the content inside the overlay (which is just a div) so we only need one `StyleMarker` and also don't need to manually specify a z-index on the modal content. The people at Radix UI do this for the dialog component they provide as part of Radix Themes, so I figured it would be okay for us 😄